pandas/tests/series/test_analytics.py

```diff
@@ -1083,6 +1083,16 @@ def test_rank_categorical(self):
         res = unordered.rank()
         assert_series_equal(res, exp_unordered)
 
+        unordered1 = pd.Series(
+            [1, 2, 3, 4, 5, 6],
+        ).astype('category').cat.set_categories(
+            [1, 2, 3, 4, 5, 6],
+            ordered=False
```

Comment:
you can simplify this creation by doing `astype('category', categories=[1, 2, 3, 4, 5, 6], ordered=False)`